extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -696,6 +696,7 @@ public static function canEarnBadges( User $user = null ) {
 			$user->isAnon() ||
 			$user->isBlocked() ||
 			( $user->isAllowed( 'bot' ) || in_array( $user->getName(), $wgWikiaBotLikeUsers ) ) ||
+			( is_array( $wgAchievementExemptUsers ) && in_array( $user->getId(), $wgAchievementExemptUsers ) ) ||
```

Comment:
If you want to avoid having to use that is_array - so code would be bit cleaner - you can just define that variable as an empty array in DefaultSettings.php